gossip/minimal/goshawk.go

```diff
@@ -83,22 +85,36 @@ func NewGoshawkFromFile(ctx context.Context, filename string, hc *http.Client, f
 
 // NewGoshawk creates a gossiper from the given configuration protobuf and optional http client.
 func NewGoshawk(ctx context.Context, cfg *configpb.GoshawkConfig, hc *http.Client, fetchOpts scanner.FetcherOptions) (*Goshawk, error) {
-	if cfg.DestHub == nil {
+	if len(cfg.DestHub) == 0 {
 		return nil, errors.New("no destination hub config found")
 	}
-	if cfg.SourceLog == nil || len(cfg.SourceLog) == 0 {
+	if len(cfg.SourceLog) == 0 {
 		return nil, errors.New("no source log config found")
 	}
 	if cfg.BufferSize < 0 {
 		return nil, fmt.Errorf("negative STH buffer size (%d) specified", cfg.BufferSize)
 	}
 
-	dest, err := hubScannerFromProto(cfg.DestHub, hc)
-	if err != nil {
-		return nil, fmt.Errorf("failed to parse dest hub config: %v", err)
+	hawk := Goshawk{
+		dests:     make(map[string]*hubScanner),
+		origins:   make(map[string]*originLog),
+		fetchOpts: fetchOpts,
+	}
+	hawk.fetchOpts.Continuous = true
```

Comment:
I think you can just set fetchOpts as it's passed by value but that might be less clear to readers. You could add .WithXXX to the fetcher stuff but I think that's probably overkill.

Reply:
Changed to set `fetchOpts`.